docs/snippets/chains/API-reference.mdx

```diff
@@ -0,0 +1,572 @@
+# API Reference
+
+## *class* `truss_chains.Assets`
+
+Specifies which assets a chainlet can access in the remote deployment.
```

Comment:
might be nice to add a "for instance, cached weights or secrets" to this sentence

Reply:
Will do.